src/SqlProviderBase.ts

```diff
@@ -208,10 +208,13 @@ export abstract class SqlProviderBase extends NoSqlProvider.DbProvider {
 
                         dropQueries = _.map(tableNames, name => trans.runQuery('DROP TABLE ' + name));
 
-                        // Drop all existing metadata
-                        const metaToDropArg = _.map(indexMetadata, meta => meta.key).join(', ');
-                        dropQueries.push(trans.runQuery('DELETE FROM metadata where name in (?) ', [metaToDropArg]));
```

Comment:
this didn't actually delete anything, changed the syntax to match other queries where we use (?)